Tasks/UseDotNetV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Use .NET Core",
-  "loc.helpMarkDown": "[Learn more about this task](https://aka.ms/AA4xgy0)",
```

Comment:
We should revert back this change, i'll update the shortlink to point to the docs url

Reply:
corrected in task.json, the generated files got autocorrected.